### PR TITLE
Added Milestones and minor changes.

### PR DESCRIPTION
**Pull request by bhattabhi013**
## Changes proposed

Added new milestones to my profile. 
Added bifurcation/ groups to the links section.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

I've added milestones to my profile.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4465"></a>

### CODE REVIEW

**Review by priyankarpal — CHANGES_REQUESTED** [review]
Hey, there is a conflict please resolve it.

**Review by priyankarpal — APPROVED** [review]
LGTM ⚡

**Comment by bhattabhi013** [comment]
> Hey, there is a conflict please resolve it.

Thanks for responding. Conflicts resolved.

File: data/bhattabhi013.json [hunk 1]
```diff
@@ -1,68 +1,105 @@
...
-      "description": "Launched my first mobile application on playstore.",
-      "url": "https://play.google.com/store/apps/details?id=com.plasma.covid"
-    }
-  ]
-}
+    "name": "Abhishek Bhatt",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Full-Stack developer | Flutter | NodeJs | Angular",
+    "avatar": "https://github.com/bhattabhi013.png",
```
Comment (stevenovak123) [inline data/bhattabhi013.json:6]: Avatar can be removed as it is automatically pickedup from GitHub

**Review by stevenovak123 — CHANGES_REQUESTED** [review]

File: data/bhattabhi013.json [hunk 2]
```diff
@@ -1,68 +1,104 @@
...
+      { "platform": "twitter", "url": "https://twitter.com/AbhishekBtwt" },
+      { "platform": "github", "url": "https://github.com/bhattabhi013" },
+      { "platform": "instagram", "url": "https://www.instagram.com/bwith_abhi" }
+    ],
+    "links": [
+      {
+        "group": "Socials",
+        "name": "GitHub: Collaborate on projects",
+        "url": "https://github.com/bhattabhi013",
+        "icon": "github"
```
Comment (priyankarpal) [inline data/bhattabhi013.json:17]: AAd this GitHub icon
``` suggestion
        "icon": "SiGithub"
```

File: data/bhattabhi013.json [hunk 3]
```diff
@@ -1,68 +1,104 @@
...
+      },
+      {
+        "group": "Socials",
+        "name": "Twitter: Follow me",
+        "url": "https://twitter.com/AbhishekBtwt",
+        "icon": "twitter"
```
Comment (priyankarpal) [inline data/bhattabhi013.json:23]: Add this Twitter icon
``` suggestion
        "icon": "SiTwitter"
```

File: data/bhattabhi013.json [hunk 4]
```diff
@@ -1,68 +1,104 @@
...
+      },
+      {
+        "group": "Socials",
+        "name": "LinkedIn: Let's connect",
+        "url": "https://www.linkedin.com/in/bhatt-abhi/",
+        "icon": "linkedin"
```
Comment (priyankarpal) [inline data/bhattabhi013.json:29]: Add this LinkedIn icon
``` suggestion
        "icon": "SiLinkedin"
```

File: data/bhattabhi013.json [hunk 5]
```diff
@@ -1,68 +1,104 @@
...
+        "group": "Open Source Projects",
+        "name": "Open Food Facts (Contributor)",
+        "url": "https://github.com/openfoodfacts/smooth-app",
+        "icon": "FaGithub"
+      },
+      {
+        "group": "Play Store",
+        "name": "Deployed Apps on Play Store",
+        "url": "https://play.google.com/store/apps/developer?id=Abhi+Bhatt",
+        "icon": "FaAndroid"
```
Comment (priyankarpal) [inline data/bhattabhi013.json:50]:
``` suggestion
        "icon": "FaGooglePlay"
```

File: data/bhattabhi013.json [hunk 6]
```diff
@@ -1,68 +1,104 @@
...
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Participated in Social Winter of Code as Project Maintainer",
+        "date": "Winter 2022",
+        "icon": "globe",
```
Comment (priyankarpal) [inline data/bhattabhi013.json:57]:
```suggestion
        "icon": "FaGlobe",
```

File: data/bhattabhi013.json [hunk 7]
```diff
@@ -1,68 +1,104 @@
...
+        "color": "black",
+        "description": "Participated as Project Maintainer to help the people get into open-source.",
+        "url": "https://certificate.givemycertificate.com/c/3ec367f5-68b7-4510-b011-2645f782fbd4"
+      },
+      {
+        "title": "Mentored at first hackathon.",
+        "date": "December 2022",
+        "icon": "globe",
```
Comment (priyankarpal) [inline data/bhattabhi013.json:65]:
```suggestion
        "icon": "FaGlobe",
```

File: data/bhattabhi013.json [hunk 8]
```diff
@@ -1,68 +1,104 @@
...
+        "color": "white",
+        "description": "I was invited as a mentor at Hack With MAIT 3.0",
+        "url": "https://twitter.com/QuasarReeha/status/1606619819646013441?s=20&t=YDAxaPCs9koMA2Q_x1biNg"
+      },
+      {
+        "title": "Vritant App",
+        "date": "Nov 2022",
+        "icon": "globe",
```
Comment (priyankarpal) [inline data/bhattabhi013.json:73]:
```suggestion
        "icon": "FaMedapps",
```

File: data/bhattabhi013.json [hunk 9]
```diff
@@ -1,68 +1,104 @@
...
+        "color": "white",
+        "description": "It makes learning fun and helps kid learn through AR and 3-D models.",
+        "url": "https://github.com/bhattabhi013/Vritant"
+      },
+      {
+        "title": "Participated in Social Summer of Code as Contributor",
+        "date": "Summer 2022",
+        "icon": "globe",
```
Comment (priyankarpal) [inline data/bhattabhi013.json:81]:
```suggestion
        "icon": "FaGlobe",
```

File: data/bhattabhi013.json [hunk 10]
```diff
@@ -1,68 +1,104 @@
...
+        "color": "black",
+        "description": "Participated in A 3 months long open-source program.",
+        "url": "https://certificate.givemycertificate.com/c/a3c3ee4b-2b69-4517-9956-b2d63b5549d6"
+      },
+      {
+        "title": "Started Developing and open-source",
+        "date": "March 2022",
+        "icon": "github",
```
Comment (priyankarpal) [inline data/bhattabhi013.json:89]:
```suggestion
        "icon": "SiGithub",
```

File: data/bhattabhi013.json [hunk 11]
```diff
@@ -1,68 +1,104 @@
...
+        "color": "black",
+        "description": "One of my best and life changing decision ever.",
+        "url": "https://github.com/bhattabhi013"
+      }, 
+      {
+        "title": "Launched first mobile application.",
+        "date": "May 2021",
+        "icon": "android",
```
Comment (priyankarpal) [inline data/bhattabhi013.json:97]:
```suggestion
        "icon": "FaGooglePlay",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
Fix these problems

**Comment by bhattabhi013** [comment]
> Fix these problems

Hi, Issue resolved

**Review by priyankarpal — DISMISSED** [review]
LGTM 👍🏻

**Comment by priyankarpal** [comment]
> Hi, Issue resolved

Done

File: data/bhattabhi013.json [hunk 12]
```diff
@@ -1,68 +1,104 @@
...
-}
+    "name": "Abhishek Bhatt",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Full-Stack developer | Flutter | NodeJs | Angular | React",
+    "tags": ["Flutter", "Node,js", "Angular", "Javascript", "Typescript", "open-source"],
+    "socials": [
+      { "platform": "twitter", "url": "https://twitter.com/AbhishekBtwt" },
+      { "platform": "github", "url": "https://github.com/bhattabhi013" },
+      { "platform": "instagram", "url": "https://www.instagram.com/bwith_abhi" }
```
Comment (Panquesito7) [inline data/bhattabhi013.json:10]: Should be `icon`. Not sure `platform` is valid?
```suggestion
      { "platform": "FaTwitter", "url": "https://twitter.com/AbhishekBtwt" },
      { "platform": "FaGithub", "url": "https://github.com/bhattabhi013" },
      { "platform": "FaInstagram", "url": "https://www.instagram.com/bwith_abhi" }
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/bhattabhi013.json [hunk 13]
```diff
@@ -1,68 +1,104 @@
...
-}
+    "name": "Abhishek Bhatt",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Full-Stack developer | Flutter | NodeJs | Angular | React",
+    "tags": ["Flutter", "Node,js", "Angular", "Javascript", "Typescript", "open-source"],
+    "socials": [
+      { "platform": "FaTwitter", "url": "https://twitter.com/AbhishekBtwt" },
+      { "platform": "FaGithub", "url": "https://github.com/bhattabhi013" },
+      { "platform": "FaInstagram", "url": "https://www.instagram.com/bwith_abhi" }
```
Comment (Panquesito7) [inline data/bhattabhi013.json:10]:
```suggestion
      { "icon": "FaTwitter", "url": "https://twitter.com/AbhishekBtwt" },
      { "icon": "FaGithub", "url": "https://github.com/bhattabhi013" },
      { "icon": "FaInstagram", "url": "https://www.instagram.com/bwith_abhi" }
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by stevenovak123 — APPROVED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**